tools/ansible/tasks/wsu/templates/cni.j2

```diff
@@ -0,0 +1,32 @@
+
+{
+    "cniVersion":"0.2.0",
+    "name":"OpenShiftNetwork",
+        "type":"win-overlay",
+ "capabilities": {
+        "dns": true
+    },
+    "ipam": {
+        "type": "host-local",
+        "subnet": "{{ ovn_host_subnet.stdout }}"
+    },
+        "policies":[
+            {
+                "name":"EndpointPolicy",
+                "value":{
+                    "Type":"OutBoundNAT",
+                    "ExceptionList":[
+                        "10.132.0.0/16"
+                    ]
+                }
+            },
+            {
+                "name":"EndpointPolicy",
+                "value":{
+                    "Type":"ROUTE",
+                    "DestinationPrefix":"10.132.0.0/16",
```

Comment:
This cant be hardcoded: "The destination prefix would come from the cluster network config objects service CIDR"

Reply:
Hmm, is it, I thought we just need to update `subnet` as per our [doc](https://docs.google.com/document/d/1zAidTs8wbWHzamh4G3pwdaPomyT-doSbo9xTAcIK9no/edit#)